doc/developers/advanced_installation.rst

```diff
@@ -552,6 +460,8 @@ the base system and these steps will not be necessary.
 
 .. _OpenMP: https://en.wikipedia.org/wiki/OpenMP
 .. _Cython: https://cython.org
+.. _Meson: https://mesonbuild.com/
+.. _Ninja: https://ninja-build.org/
```

Comment:
This does not seem to be used anywhere but I think we should find a way to explain somewhere that meson relies on ninja under the hood to build native extensions.

Reply:
I think the two references that were removed are more likely to be useful than Meson or Ninja docs:

- `pandas setup doc
  <https://pandas.pydata.org/docs/development/contributing_environment.html#step-3-build-and-install-pandas>`_:
  pandas has a similar setup as ours (no spin or dev.py)
- `scipy Meson doc
  <https://scipy.github.io/devdocs/building/understanding_meson.html>`_ gives
  more background about how Meson works behind the scenes

Reply:
Ninja was indeed left over from when I thought you didn't have to install it explicitly. Now it is useful again because you have to install it explicitly and I list it as a build time dependency.